### PR TITLE
Fix https://github.com/pytorch/xla/pull/7679 test

### PR DESCRIPTION
**Pull request by zpcore**
Follow up fix for https://github.com/pytorch/xla/pull/7679. If this one still fail, I will revert both.

### CODE REVIEW

**Comment by JackCaoG** [comment]
can you rerun to trigger the tpu ci?

**Comment by zpcore** [comment]
How to trigger the TPU test? I pressed the rerun and TPU test still get skipped.

**Comment by will-cromar** [comment]
You need to push a new commit for the label to take effect. Try rebasing your fixes on master to clean up the commit history too

File: docs/source/index.rst [hunk 1]
```diff
@@ -56,6 +56,7 @@ xla_model
 .. autofunction:: xla_device
 .. autofunction:: xla_device_hw
 .. autofunction:: is_master_ordinal
+.. autofunction:: xrt_world_size
```
Comment (JackCaoG) [inline docs/source/index.rst:59]: why add `xrt_world_size ` here? I thought we are going to use `xr.world_size` instead

File: test/pjrt/test_runtime.py [hunk 2]
```diff
@@ -14,6 +15,8 @@
 class TestExperimentalPjrt(parameterized.TestCase):
 
   def setUp(self):
+    global xr
+    reload(xr)
```
Comment (JackCaoG) [inline test/pjrt/test_runtime.py:19]: why is this needed

File: test/pjrt/test_runtime_multi_cpu.py [hunk 3]
```diff
@@ -41,17 +41,8 @@ def test_multi_cpu_devices(self):
...
-  def test_global_ordinal(self, ordinal_func):
-    results = pjrt.run_multiprocess(ordinal_func)
-    self.assertListEqual(sorted(results.values()), [0, 1, 2, 3])
-
-  @parameterized.named_parameters(('xla_model', xm.get_local_ordinal),
-                                  ('pjrt', xr.local_ordinal))
-  def test_local_ordinal(self, ordinal_func):
-    # TODO(wcromar): add multiprocess tests
-    results = pjrt.run_multiprocess(ordinal_func)
+  def test_global_ordinal(self):
```
Comment (JackCaoG) [inline test/pjrt/test_runtime_multi_cpu.py:44]: why `test_global_ordinal ` is removed?

File: test/pjrt/test_runtime.py (hunk 2 above)
Comment (zpcore) [reply test/pjrt/test_runtime.py:19]: I added @run_once decorator for using_pjrt(), which remembers the previous call result. Without the reload, each test will impact with each other under the same test class.

**Comment by JackCaoG** [comment]
Is this pr suppose to be this big? or it is some github commit issue?

File: test/pjrt/test_runtime_multi_cpu.py (hunk 3 above)
Comment (zpcore) [reply test/pjrt/test_runtime_multi_cpu.py:44]: `xm.get_ordinal` and `xr.global_ordinal` are the same function now. I think you mean `test_local_ordinal`? I may make a mistake here. Let me bring back `xr.local_ordinal` test.

**Comment by zpcore** [comment]
> Collaborator

Let me create a new PR based on the master branch.

**Comment by zpcore** [comment]
Use this one: https://github.com/pytorch/xla/pull/7743 instead.
